java/src/main/resources/hbase-site.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<configuration>
+    <!--
+      The following properties are set for running HBase as a single process on a
+      developer workstation. With this configuration, HBase is running in
+      "stand-alone" mode and without a distributed file system. In this mode, and
+      without further configuration, HBase and ZooKeeper data are stored on the
+      local filesystem, in a path under the value configured for `hbase.tmp.dir`.
+      This value is overridden from its default value of `/tmp` because many
+      systems clean `/tmp` on a regular basis. Instead, it points to a path within
+      this HBase installation directory.
+
+      Running against the `LocalFileSystem`, as opposed to a distributed
+      filesystem, runs the risk of data integrity issues and data loss. Normally
+      HBase will refuse to run in such an environment. Setting
+      `hbase.unsafe.stream.capability.enforce` to `false` overrides this behavior,
+      permitting operation. This configuration is for the developer workstation
+      only and __should not be used in production!__
+
+      See also https://hbase.apache.org/book.html#standalone_dist
+    -->
+    <property>
+        <name>hbase.cluster.distributed</name>
+        <value>true</value>
+    </property>
+    <property>
+        <name>hbase.tmp.dir</name>
+        <value>./tmp</value>
+    </property>
+    <property>
+        <name>hbase.unsafe.stream.capability.enforce</name>
+        <value>false</value>
+    </property>
+    <property>
+        <name>hbase.zookeeper.property.initLimit</name>
+        <value>20</value>
+    </property>
+    <property>
+        <name>hbase.rootdir</name>
+        <value>hdfs://<IP-address>/hbase</value>
```

Comment:
how do we expect <IP-address> to be updated ?
This approach seems static and will not work for templates.